### ISSUE TITLE
Cannot dlopen some GPU libraries

### ISSUE THREAD

**Issue by MinWang1997** [TUI-NICR/ESANet#24]
![no_lib_error](https://user-images.githubusercontent.com/75472853/130356944-08faf7a9-ca0b-4b58-8eba-d674a97c0a3b.png)

Hi all,
I followed ReadMe for installation but I got lib errors when I train models via train.py. I have checked tf-GPU, cuda and cudnn versions. Could you help me figure it out?

**Comment by danielS91** [comment]
We are facing the same errors/warnings. It tells you that your version of TensorFlow is not compatible with the CUDA / cuDNN version installed by PyTorch. However, we do not use any CUDA / cuDNN stuff from TensorFlow. TensorFlow is only used to speed up confusion matrix calculation. Therefore, you can simply ignore these warnings/errors. 

However, if you want to get rid of these errors/warnings, you can simply switch to the CPU version of TensorFlow by replacing `tensorflow-gpu=1.15.0` with `tensorflow-cpu=1.15.0` in the [environment file](https://github.com/TUI-NICR/ESANet/blob/main/rgbd_segmentation.yaml#L120) or using pip if the environment is already created. The CPU version works well.

**Comment by MinWang1997** [comment]
> We are facing the same errors/warnings. It tells you that your version of TensorFlow is not compatible with the CUDA / cuDNN version installed by PyTorch. However, we do not use any CUDA / cuDNN stuff from TensorFlow. TensorFlow is only used to speed up confusion matrix calculation. Therefore, you can simply ignore these warnings/errors.
> 
> However, if you want to get rid of these errors/warnings, you can simply switch to the CPU version of TensorFlow by replacing `tensorflow-gpu=1.15.0` with `tensorflow-cpu=1.15.0` in the [environment file](https://github.com/TUI-NICR/ESANet/blob/main/rgbd_segmentation.yaml#L120) or using pip if the environment is already created. The CPU version works well.

Hi Daniel, thank you for your reply!
I thought I cannot run the code with errors before and now I noticed that the output file is renewed.
The tf errors disappear after loading cuda 10.0.130 and cudnn 10.1_v7.5

We can also ignore the other warning e.g., UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.

**Comment by mona0809** [comment]
Yes, since we always pass the epoch explicitly as a parameter (https://github.com/TUI-NICR/ESANet/blob/main/train.py#L264), the warning can be ignored here.

**Comment by zhouqunbing** [comment]
i replace tensorflow-gpu=1.15.0 with tensorflow-cpu=1.15.0,but when model in the test phase,the same error occurred.
Could not dlopen library 'libcufft.so.10.0'; dlerror: libcufft.so.10.0: cannot open shared object file: No such file or directory...
